release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -298,8 +298,191 @@ For more information, see xref:../openshift_images/image-streams-manage.adoc#ima
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on {ibmzProductName} and {linuxoneProductName} with {product-title} {product-version}:
+
+* Cluster Resource Override Operator
+* Egress IP
+* FIPS cryptography
+* MetalLB Operator
+* Network-Bound Disk Encryption - External Tang Server
+
+[discrete]
+==== IBM Secure Execution
+
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and {linuxoneProductName} (s390x architecture).
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution]
+
+[discrete]
+==== Supported features
+
+The following features are also supported on {ibmzProductName} and {linuxoneProductName}:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Compliance Operator
+** File Integrity Operator
+** Local Storage Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** Service Binding Operator
+** Vertical Pod Autoscaler Operator
+* The following Multus CNI plugins are supported:
+** Bridge
+** Host-device
+** IPAM
+** IPVLAN
+* Alternate authentication providers
+* Automatic Device Discovery with Local Storage Operator
+* CSI Volumes
+** Cloning
+** Expansion
+** Snapshot
+* Cron jobs
+* Descheduler
+* Encrypting data stored in etcd
+* Helm
+* Horizontal pod autoscaling
+* IPv6
+* Monitoring for user-defined projects
+* Multipathing
+* Operator API
+* OC CLI plugins
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* PodDisruptionBudget
+* OVN-Kubernetes, including IPsec encryption
+* Scheduler profiles
+* Stream Control Transmission Protocol (SCTP)
+* Support for multiple network interfaces
+* Three-node cluster support
+* z/VM Emulated FBA devices on SCSI disks
+* 4K FCP block device
+
+These features are available only for {product-title} on {ibmzProductName} and {linuxoneProductName} for {product-version}:
```

Comment:
The "only" is because HyperPAV is only supported on the Z architecture. Not on x86 or ppc64le. 
But yes collapsing is ok.

Reply:
"These features" is throwing me off. This could mean the features before and after this statement. I think "The following feature" would be better here. WDYT?

I agree with Joe. If all these features are not new to 4.13, should we be listing them? Maybe a link to feature docs would be better. From reading the `HyperPAV enabled` entry, this feature seems like the only new addition to 4.13?